### DISCUSSION TITLE
Always show remote content from .... - does not seem to work in Thunderbird 91.2?

### DISCUSSION THREAD

**Discussion by NaveenGarg** [protz/thunderbird-conversations#1728] [Q&A] [answered]
Greetings, I simply love this add-on - Thank you! However I noted that I cannot seem to click on "always show remote content from ..." to enable remote content from any particular sender to be shown by default. For now, I have to click on "show remote content.." link to the left of the above link, each time for each new message from the same sender to see the remote content. It is quite an annoyance. Is this a bug or am I missing some setting somewhere? - Thanks for your help!

**Comment by NaveenGarg** [comment, marked as answer]
Here is the error log for this.

Uncaught (in promise) TypeError: Services.perms.add is not a function
    alwaysShowRemoteContent jar:file:///home/naveengarg/.thunderbird/y01zchr9.default/extensions/gconversation@xulforum.org.xpi!/experiment-api/api.js:445
    alwaysShowRemoteContent chrome://conversations/content/stub.bundle.js:882
    createThunkMiddleware chrome://conversations/content/751.bundle.js:612
    onAlwaysShowRemote chrome://conversations/content/195.bundle.js:3390
    Rb chrome://conversations/content/415.bundle.js:245
    Xb chrome://conversations/content/415.bundle.js:245
    Yb chrome://conversations/content/415.bundle.js:246
    Ze chrome://conversations/content/415.bundle.js:293
    se chrome://conversations/content/415.bundle.js:294
    jd chrome://conversations/content/415.bundle.js:306
    Jb chrome://conversations/content/415.bundle.js:485
    Nb chrome://conversations/content/415.bundle.js:243
    jd chrome://conversations/content/415.bundle.js:298
    yc chrome://conversations/content/415.bundle.js:268
    hd chrome://conversations/content/415.bundle.js:267
    unstable_runWithPriority chrome://conversations/content/415.bundle.js:545
    gg chrome://conversations/content/415.bundle.js:315
    Hb chrome://conversations/content/415.bundle.js:485
    gd chrome://conversations/content/415.bundle.js:266

**Reply by Standard8** [reply]
Thanks for the extra info, for things like this you can create them directly as issues.

I've just fixed this in a75d7abbb2cfa629c103755eae6fe42c01f2a419. We'll get a new release posted in the next few days.

**Reply by NaveenGarg** [reply]
Awesome! Thanks for a very prompt response!!!!
